terraform/groups/ecs-service/locals.tf

```diff
@@ -2,85 +2,76 @@
 locals {
   stack_name                = "order-service" # this must match the stack name the service deploys into
   name_prefix               = "${local.stack_name}-${var.environment}"
+  global_prefix             = "global-${var.environment}"
   service_name              = "admin-orders-web"
   container_port            = "3000" # default node port required here until prod docker container is built allowing port change via env var
+  eric_port                 = "10000"
   docker_repo               = "admin.orders.web.ch.gov.uk"
+  kms_alias                 = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority = 9
   lb_listener_paths         = ["/orders-admin/*"]
   healthcheck_path          = "/orders-admin" #healthcheck path for confirmation statement web
-  healthcheck_matcher       = "200-302" # no explicit healthcheck in this service yet, change this when added!
-
-  kms_alias                 = "alias/${var.aws_profile}/environment-services-kms"
-  service_secrets           = jsondecode(data.vault_generic_secret.service_secrets.data_json)
-
-  parameter_store_secrets    = {
-    "vpc_name"                  = local.service_secrets["vpc_name"]
-    "chs_api_key"               = local.service_secrets["chs_api_key"]
-    "internal_api_url"          = local.service_secrets["internal_api_url"]
-    "cdn_host"                  = local.service_secrets["cdn_host"]
-    "oauth2_auth_uri"           = local.service_secrets["oauth2_auth_uri"]
-    "oauth2_redirect_uri"       = local.service_secrets["oauth2_redirect_uri"]
-    "account_url"               = local.service_secrets["account_url"]
-    "cache_server"              = local.service_secrets["cache_server"]
-  }
-
-  vpc_name                  = local.service_secrets["vpc_name"]
-  chs_api_key               = local.service_secrets["chs_api_key"]
-  internal_api_url          = local.service_secrets["internal_api_url"]
-  cdn_host                  = local.service_secrets["cdn_host"]
-  oauth2_auth_uri           = local.service_secrets["oauth2_auth_uri"]
-  oauth2_redirect_uri       = local.service_secrets["oauth2_redirect_uri"]
-  account_url               = local.service_secrets["account_url"]
-  cache_server              = local.service_secrets["cache_server"]
+  healthcheck_matcher       = "200-302"       # no explicit healthcheck in this service yet, change this when added!
```

Comment:
The terraform module 'healthcheck_matcher' can only take one HTTP response code.